src/server/rest/v1/service/BillingService.ts

```diff
@@ -625,7 +624,7 @@ export default class BillingService {
     UtilsService.assertObjectExists(action, user, `User ID '${userID}' does not exist`,
       MODULE_NAME, 'handleBillingDeletePaymentMethod', req.user);
     // Invoke the billing implementation
-    await billingImpl.deletePaymentMethod(user, filteredRequest.paymentMethodId);
+    const operationResult: BillingOperationResult = await billingImpl.deletePaymentMethod(user, filteredRequest.paymentMethodId);
```

Comment:
No need to put the type BillingOperationResult here, it is indirectly deduced by the return param of the method.